src/lib.rs

```diff
@@ -255,23 +293,6 @@ pub fn describe_eif(eif_path: String) -> NitroCliResult<DescribeEifInfo> {
     Ok(info)
 }
 
-/// Returns the value of the `NITRO_CLI_BLOBS` environment variable.
-///
-/// This variable specifies where all the blobs necessary for building
-/// an enclave image are stored. As of now the blobs are:
-/// - *bzImage*: A kernel image if the local arch is x86_64 or
-/// - *Image*  : A kernel image if the local arch is aarch64
-/// - *init*: The initial init process that is bootstraping the environment.
-/// - *linuxkit*: A slightly modified version of linuxkit.
-/// - *cmdline*: A file containing the kernel commandline.
-fn blobs_path() -> NitroCliResult<String> {
-    // TODO Improve error message with a suggestion to the user
-    // consider using the default path used by rpm install
-    let blobs_res = std::env::var("NITRO_CLI_BLOBS");
```

Comment:
I don't think we can drop reading from this env variable if we want to upstream these changes.

Reply:
Changed the implementation to make sure the resource directory is located in the following order:
- The path passed to `build_from_docker`
- The path located in `NITRO_CLI_BLOB`
- The default location